src/ai/backend/manager/api/scaling_group.py

```diff
@@ -93,13 +93,21 @@ async def get_wsproxy_version(request: web.Request, params: Any) -> web.Response
                 if not wsproxy_addr:
                     wsproxy_version = "v1"
                 else:
-                    wsproxy_status = await query_wsproxy_status(wsproxy_addr)
-                    wsproxy_version = wsproxy_status["api_version"]
-                return web.json_response(
-                    {
-                        "wsproxy_version": wsproxy_version,
-                    }
-                )
+                    try:
+                        wsproxy_status = await query_wsproxy_status(wsproxy_addr)
+                        wsproxy_version = wsproxy_status["api_version"]
+                        return web.json_response(
+                            {
+                                "wsproxy_version": wsproxy_version,
+                            }
+                        )
+                    except aiohttp.ClientConnectorError:
+                        log.error(
+                            "Failed to query the wsproxy {1} configured for sg:{0}",
+                            scaling_group_name,
+                            wsproxy_addr,
+                        )
+                        return URLNotFound()
```

Comment:
Hm... sorry for requesting changes here again, but returning `URLNotFound` would be indistiguishable from the absence of `get_wsproxy_version` API itself.

Since this exception block is executed when the wsproxy address is invalid or inaccessible, I think we should wrap this using `ServerMisconfiguredError`.

Reply:
Thanks for reviewing!
I will fix it.
And I fixed web ui code too.